### ISSUE TITLE
App not accessible with the VoiceOver screen reader on MacOS

### ISSUE THREAD

**Issue by ArsGev25** [Anjok07/ultimatevocalremovergui#1984]
I’m totally blind, and use VoiceOver to access my computer. I tried downloading the latest version of the app, as well as the roformer beta. But neither of these work with the screen reader at all. I tried OCR, but despite it helping with a few things, it’s still not really efficient. I’d appreciate if the people working on this app would pay attention to its accessibility, because for stem separation, there’s a lot of potential with all the models incorporated. I wouldn’t want the accessibility to be left behind also.

**Comment by deton24** [comment]
You could switch to MSST-GUI instead. It's compatible with screen reader:
https://github.com/ZFTurbo/Music-Source-Separation-Training/blob/main/docs/gui.md

**Comment by ArsGev25** [comment]
> You could switch to MSST-GUI instead. It's compatible with screen reader: https://github.com/ZFTurbo/Music-Source-Separation-Training/blob/main/docs/gui.md

Are the roformer models supported with that? I’m a bit new to this

**Comment by deton24** [comment]
Yes. Only the old MDX-Net models like HQ_4 and 5 are not supported, but
MDX23C and Roformers are.

pon., 29 wrz 2025 o 23:37 ArsGev25 ***@***.***> napisał(a):

**Comment by ArsGev25** [comment]
> Yes. Only the old MDX-Net models like HQ_4 and 5 are not supported, but
> MDX23C and Roformers are.
> 
> pon., 29 wrz 2025 o 23:37 ArsGev25 ***@***.***> napisał(a):
> […](#)

One more question. I see in the UVR GUI there are certain models (or should I say variants of models) not present in the MSST GUI. The one I remember off the top of my head now is htdemucsv4_mmi, sorry if I get the name a bit wrong. Would these be available to use through command line in some way? Appreciate you
